lib/active_interactor/interactor/result.rb

```diff
@@ -113,10 +113,18 @@ def method_missing(method_name, *args, &block)
         super
       end
 
-      def resolve_errors!
-        all_errors = context.errors.uniq.compact
-        context.errors.clear
-        all_errors.each { |error| context.errors.add(error[0], error[1]) }
+      if ActiveModel::Errors.method_defined?(:import)
```

Comment:
I would rather see this split into two methods to ensure we don't inflate the complexity of `resolve_errors!`

Reply:
additionally we can call this in the method directly instead with `context.errors.respond_to?`

Reply:
```ruby
def resolve_errors!
  all_errors = context.errors.uniq.compact
  context.errors.clear
  merge_errors(all_errors)
end

def merge_errors(errors)
  errors.each do |error|
    if error.is_a?(ActiveModel::Error)
      context.errors.import(error)
    else
      context.errors.add(error[0], error[1])
    end
  end
end
```

Reply:
lastly it looks like we can possibly get rid of this logic all together by just calling `context.errors.uniq!`